pepys_admin/utils.py

```diff
@@ -187,23 +187,30 @@ def database_at_latest_revision(db_path):
         )
         return False
 
+    versions = read_latest_revisions_file()
+    if len(versions) == 0:
+        return
```

Comment:
@robintw - the other `return` statements all include a boolean. Should we return `False` here?